plugins/pom.xml

```diff
@@ -341,8 +341,8 @@
                                 <jspc>
                                     <package>org.jivesoftware.openfire.plugin.${plugin.name}</package>
                                 </jspc>
-                                <sourceVersion>11</sourceVersion>
-                                <targetVersion>11</targetVersion>
+                                <sourceVersion>17</sourceVersion>
+                                <targetVersion>17</targetVersion>
```

Comment:
This change shouldn't be necessary, as Openfire's HEAD of the Main branch already has this change.

Can you rebase the PR on the latest HEAD of the Main branch of igniterealtime/Openfire please? That should take away the need for this change (and possibly others).

Reply:
I stand corrected. This was _not_ in the HEAD of the Main branch yet (we've missed it). I would still like to see that Java-upgrade change be isolated from the Jetty upgrade change. I've pulled out this change in https://github.com/igniterealtime/Openfire/pull/2601